specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,2921 @@
+                  }
+                },
+                "default": {
+                  "description": "Unexpected error",
+                  "schema": {
+                    "$ref": "#/definitions/ErrorResponse"
+                  }
+                }
+              },
+              "x-ms-examples": {
+                "Successful Get Text Analysis Job Status Request": {
+                  "$ref": "./examples/SuccessfulAnalyzeTextJobStatusRequest.json"
+                }
+              },
+              "deprecated": false
+            },
+            "delete": {
+                "produces": [
+                  "application/json",
+                  "text/json"
+                ],
+                "description": "Cancel analysis job.",
+                "operationId": "AnalyzeText_CancelJob",
+                "summary": "Cancel analysis job",
+                "parameters": [
+                  {
+                    "$ref": "#/parameters/JobId"
+                  }
+                ],
+                "responses": {
+                  "202": {
+                    "description": "Cancel Job request has been received.",
+                    "headers": {
+                      "Operation-Location": {
+                        "type": "string"
+                      }
+                    }
+                  },
+                  "default": {
+                    "description": "Unexpected error",
+                    "schema": {
+                      "$ref": "#/definitions/ErrorResponse"
+                    }
+                  }
+                },
+                "x-ms-examples": {
+                  "Successful Delete Text Analysis Job Request": {
+                    "$ref": "./examples/SuccessfulAnalyzeTextJobDeleteRequest.json"
+                  }
+                },
+                "x-ms-long-running-operation": true
+              }
+          }
+    },
+    "definitions": {
+        "AnalyzeTextTaskKind": {
+            "type": "string",
+            "description": "Enumeration of supported Text Analysis tasks.",
+            "enum": [
+                "SentimentAnalysis",
+                "EntityRecognition",
+                "PiiEntityRecognition",
+                "KeyPhraseExtraction",
+                "EntityLinking",
+                "Healthcare",
+                "ExtractiveSummarization",
+                "CustomEntityRecogntion",
+                "CustomSingleClassification",
+                "CustomMultiClassification"
+            ],
+            "x-ms-enum": {
+                "modelAsString": true
+            }
+        },
+        "AnalyzeTextInput": {
+            "allOf": [
+                {
+                    "$ref": "#/definitions/MultiLanguageBatchInput"
+                },
+                {
+                    "$ref": "#/definitions/AnalyzeTextTask"
+                }
+            ]
+        },
+        "AnalyzeTextTask": {
+            "description": "TODO",
```

Comment:
Eh, `TODO`?

Reply:
Added more description here and in a few other places as well.